File: app/src/am_sangntn/com/example/asian/model/MetaData.java
package com.example.asian.model;

import com.google.gson.annotations.SerializedName;

public class MetaData {
    private String app;
    private String title;
    private String url;
    private String desc;

    @SerializedName("original_title")
    private String originalTitle;

    @SerializedName("original_url")
    private String originalUrl;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }
}

File: app/src/main/java/com/example/ex02/bean/Metadata.kt
package com.example.ex02.bean

import com.google.gson.annotations.SerializedName

data class Metadata(
    val app: String,

    val title: String,

    val url: String,

    val desc: String,

    @SerializedName("original_title")
    val originalTitle: String,

    @SerializedName("original_url")
    val originalUrl: String
)